amundsen_application/static/js/components/TableDetail/RequestMetadataForm/index.tsx

```diff
@@ -91,26 +99,29 @@ export class RequestMetadataForm extends React.Component<RequestMetadataProps, R
     const form = document.getElementById("RequestForm") as HTMLFormElement;
     const formData = new FormData(form);
     const recipientString = formData.get('recipients') as string
-    const recipients = recipientString.split(",")
+    const recipients = recipientString.split(RECIPIENT_LIST_DELIMETER)
     const sender = formData.get('sender') as string;
     const descriptionRequested = formData.get('table-description') === "on";
     const fieldsRequested = formData.get('column-description') === "on";
     const comment = formData.get('comment') as string;
+    const { cluster, database, schema, table_name } = this.props.tableMetadata;
     this.props.submitNotification(
       recipients,
       sender,
       NotificationType.METADATA_REQUESTED,
       {
         comment,
         resource_name: this.props.displayName,
-        resource_url: window.location.href,
+        resource_path: `/table_detail/${cluster}/${database}/${schema}/${table_name}`,
```

Comment:
Given the discrepancies between the schema we get from the search service and the schema we get from the metadata service, I opted not to write that method as since it would have more branching logic than common logic. 
Yet another reason to unify the schemas sooner rather than later to make the code on the frontend more maintainable.

Reply:
I see. 

If I could redo this logic, the URL would just be `table_details?key=${table key}` and then we'd never need to deal with the whole match params and lengthy url building.

Reply:
We can still do this if the team agree and we ever have opportunity to prioritize it or do some tech debt work. We would just have to write some kind of migration support code so that old url patterns convert to new url patterns.